tests/security/yast2_apparmor/settings_disable_enable_apparmor.pm

```diff
@@ -24,19 +24,22 @@ sub run {
     send_key "alt-e";
     assert_screen("AppArmor-Settings-Disable-Apparmor");
     wait_screen_change { send_key "alt-q" };
+    type_string("\n");
+    send_key "ctrl-c";
```

Comment:
Lili, 

I could not understand why do these operations, can you pls add some comments there?

Reply:
Good and hard question.
See links
https://openqa.suse.de/tests/7918741#step/settings_disable_enable_apparmor/27 
https://openqa.suse.de/tests/7135770#step/settings_disable_enable_apparmor/27
These are examples show sometimes the cursor was lost or the console is not clean so just try to type a character or do ctrl-c make console be ready.